build.gradle

```diff
@@ -202,6 +202,22 @@ model {
     }
 }
 
+project(':devApp') {
+    apply plugin: 'java'
+    apply plugin: 'application'
+
+    dependencies {
+        compile project(':')
+        runtime files(project(':').nativeTestFilesJar.archivePath)
+    }
+
+    sourceSets.main.java.srcDirs = [ "${rootDir}/src/dev/java" ]
```

Comment:
Because we decided we wanted consistency between c++ and Java, so that's the folder layout we wanted. It's just a test app for developers anyway. Other people didn't like the subprojects idea for CPP.